client/my-sites/checkout/src/hooks/use-should-collapse-last-step.tsx

```diff
@@ -0,0 +1,8 @@
+import { hasCheckoutVersion } from '@automattic/wpcom-checkout';
+
+export function useShouldCollapseLastStep(): boolean {
+	if ( hasCheckoutVersion( 'collapse-steps' ) ) {
+		return true;
+	}
+	return false;
+}
```

Comment:
This is great, but what about simplifying it like this:

```
export function useShouldCollapseLastStep(): boolean {
  return hasCheckoutVersion('collapse-steps');
}
```

Reply:
That's true. I left it obtuse like that because it's going to need to change again for the AB test (we'll need to add a `useExperiment` call – that's also why it's in a React hook).